clients/vault/src/oracle/agent.rs

```diff
@@ -57,82 +84,76 @@ async fn handle_message(
 	Ok(())
 }
 
-/// Start the connection to the Stellar Node.
-/// Returns an `OracleAgent` that will handle incoming messages from Stellar Node,
-/// and to send messages to Stellar Node
-pub async fn start_oracle_agent(
```

Comment:
Renamed to `listen_for_stellar_messages(...)` to return nothing. This is an added task to be monitored in the tokio_metrics